platform/o.n.bootstrap/src/org/netbeans/TopSecurityManager.java

```diff
@@ -19,163 +19,13 @@
 
 package org.netbeans;
 
-import java.awt.AWTPermission;
-import java.io.FileDescriptor;
-import java.lang.reflect.Field;
-import java.lang.reflect.Method;
-import java.net.InetAddress;
-import java.net.URL;
-import java.net.UnknownHostException;
-import java.security.AccessControlContext;
-import java.security.AccessController;
-import java.security.AllPermission;
-import java.security.CodeSource;
-import java.security.Permission;
-import java.security.PrivilegedActionException;
-import java.security.PrivilegedExceptionAction;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Set;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-import java.util.logging.LoggingPermission;
-import org.openide.util.Lookup;
-import org.openide.util.WeakSet;
-
 /** NetBeans security manager implementation.
 * @author Ales Novak, Jesse Glick
```

Comment:
I think my name can be removed now that it has been gutted. :grin:

Reply:
it would be rude to remove author information! :) This class has only one method left, we probably should move it somewhere else in another pass - I kept it minimal for now.

Reply:
renamed the class to `NbExit` since the only remaining purpose seems to be to not block shutdown hooks which might call exit() too.

I reimplemented the logic with `AtomicBoolean` to remove a race condition + removed author information as requested.